src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoTableMapping.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.azure.kusto.kafka.connect.sink;
+
+import org.apache.kafka.common.config.ConfigException;
+
+import java.util.Objects;
+
+public class KustoTableMapping {
```

Comment:
rename please, you can't understand from the name what is mapped to what...
TopicToTableMapping
or 
TopicTableWriteOptions

Reply:
Renamed to TopicToTableMapping